<ui version="4.0" >
 <class>TokenHelpWidget</class>
 <widget class="QWidget" name="TokenHelpWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>KRename Help</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>All supported tokens (commands) are listed in this dialog. You can search for tokens and insert a found token into the pattern for the filename.</string>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="KTreeWidgetSearchLineWidget" name="searchCategory" />
       </item>
       <item>
        <widget class="QTreeWidget" name="listCategories" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <column>
          <property name="text" >
           <string>Category</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="KTreeWidgetSearchLineWidget" name="searchToken" />
       </item>
       <item>
        <widget class="QTreeWidget" name="listTokens" >
         <property name="alternatingRowColors" >
          <bool>true</bool>
         </property>
         <property name="rootIsDecorated" >
          <bool>false</bool>
         </property>
         <property name="itemsExpandable" >
          <bool>false</bool>
         </property>
         <property name="sortingEnabled" >
          <bool>true</bool>
         </property>
         <property name="allColumnsShowFocus" >
          <bool>true</bool>
         </property>
         <property name="headerHidden" >
          <bool>false</bool>
         </property>
         <column>
          <property name="text" >
           <string>Token</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Description</string>
          </property>
         </column>
         <column>
          <property name="text" >
           <string>Preview</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QCheckBox" name="checkPreview" >
       <property name="text" >
        <string>Token &amp;Preview</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPreview" >
       <property name="text" >
        <string>Preview File:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboPreview" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="sizeIncrement" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTreeWidgetSearchLineWidget</class>
   <extends>QWidget</extends>
   <header>ktreewidgetsearchline.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
